tests/test_cli_compile.py

```diff
@@ -333,8 +333,9 @@ def test_local_url_package(
         cli, ["-n", "--rebuild"] + (["--generate-hashes"] if generate_hashes else [])
     )
     assert out.exit_code == 0
-    assert rewritten_line in out.stderr
-    assert dependency in out.stderr
+    output_lines = [line.strip('\\').strip() for line in out.stderr.splitlines()]
```

Comment:
output has ` \` at the end of some lines. So I remove them first.

Reply:
Ah, forget it. This test is more complicated. Could you revert it to 1687aac6e45d83204d6a5172591b8a79ec70015e back?

Reply:
Ok, reverted.

Reply:
Thanks! Sorry fo that.